### ISSUE TITLE
Error with imagemagick 7

### ISSUE THREAD

**Issue by last-partizan** [jazzband/sorl-thumbnail#591]
When using imagemagick 7 (via wand engine) to create thumbnail for png image with alpha channel i get following error:
```
ValueError: type must be defined in wand.image.IMAGE_TYPES, not 'truecolormatte'
```

This is becouse here sorl assumes `truecolormatte` as acceptable value, but it is working only in imagemagick-6, for 7 there must be `truecoloralpha` (see https://github.com/emcconville/wand/blob/master/wand/image.py#L7732).

And here must be similar check:
https://github.com/jazzband/sorl-thumbnail/blob/56e6a97780ded945e1081ace6505ee46332a5687/sorl/thumbnail/engines/wand_engine.py#L57

Can someone fix it? or should i make pull-prequest?

**Comment by HoshiYamazaki** [comment]
I have the same issue.

**Comment by fredDesign** [comment]
fix it please ...

**Comment by jaradc** [comment]
Same. Did anyone else solve this yet?

**Comment by mwschall** [comment]
This [pull request](https://github.com/jazzband/sorl-thumbnail/pull/648) should do it as per Wand [recommends](https://docs.wand-py.org/en/0.6.2/whatsnew/0.5.html#alpha-replaces-opacity-matte). There may still be other [6 -> 7](https://www.imagemagick.org/script/porting.php) incompatibilities lurking though.

**Comment by claudep** [comment]
Closed by #648. Open new tickets if other incompatibilities are found.
